src/app/StoreProvider.jsx

```diff
@@ -1,9 +1,9 @@
 'use client'
-import { useRef } from 'react'
+
 import { Provider } from 'react-redux'
-import { store } from '../Redux/Store.js';
+import { store } from '../Redux/Store';
 
 export default function StoreProvider({ children }) {
 
   return <Provider store={store}>{children}</Provider>
-}
\ No newline at end of file
+} 
```

Comment:
EOF - end of file, there should be an empty line in the end of file

Reply:
Ok sir